AgentDiscoveries-Backend/src/main/resources/config.properties

```diff
@@ -1,20 +0,0 @@
-

```

Comment:
Guessing you didn't mean to check this in - looks like something weird had happened with git when we were looking at this file. Can we undo this?

Reply:
I am not sure how to undo this. (confused face)